client/web/src/search/results/streaming/progress/StreamingProgressSkippedPopover.tsx

```diff
@@ -6,8 +6,27 @@ import { Alert, Button, Form, FormGroup, Input, Label } from 'reactstrap'
 import { Markdown } from '../../../../../../shared/src/components/Markdown'
 import { renderMarkdown } from '../../../../../../shared/src/util/markdown'
 import { SyntaxHighlightedSearchQuery } from '../../../../components/SyntaxHighlightedSearchQuery'
+import { Skipped } from '../../../stream'
 import { StreamingProgressProps } from './StreamingProgress'
 
+const sortBySeverity = (a: Skipped, b: Skipped): number => {
+    const severityToNumber = (severity: Skipped['severity']): number => {
+        switch (severity) {
+            case 'error':
+                return 1
+            case 'warn':
+                return 2
+            case 'info':
+                return 3
+        }
+    }
+
+    const aSev = severityToNumber(a.severity)
+    const bSev = severityToNumber(b.severity)
+
+    return aSev - bSev
+}
```

Comment:
```suggestion
const severityToNumber = (severity: Skipped['severity']): number => {
    switch (severity) {
        case 'error':
            return 1
        case 'warn':
            return 2
        case 'info':
            return 3
    }
}

const sortBySeverity = (a: Skipped, b: Skipped): number => {

    const aSev = severityToNumber(a.severity)
    const bSev = severityToNumber(b.severity)

    return aSev - bSev
}
```

Very nit: I would move `severityToNumber()` to an outer scope as it doesn't close over anything.